index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>miracuStore  | home</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://miraculib.netlify.app/main.css">
+            
+    <script src="https://kit.fontawesome.com/07628739a6.js" crossorigin="anonymous"></script>
+</head>
+<body>
+
+        <nav class="navbar box-shadow">
+            <p class="brand-name flex"><a href="index.html">
```

Comment:
I think its not semantically correct to wrap the brand name(logo) in a ```<p>``` . Instead you can wrap the entire logo name and icon in a single ```<h2>``` or better ```<h1>```